src/ClientHandle.cpp

```diff
@@ -77,11 +77,11 @@ cClientHandle::cClientHandle(const AString & a_IPString, int a_ViewDistance) :
 	m_BlockDigAnimStage(-1),
 	m_BlockDigAnimSpeed(0),
 	m_BlockDigAnimX(0),
-	m_BlockDigAnimY(256),  // Invalid Y, so that the coords don't get picked up
+	m_BlockDigAnimY(cChunkDef::Height + 1),  // Invalid Y, so that the coords don't get picked up
```

Comment:
@sphinxc0re The actual value assigned doesn't make much difference, as long as it's an invalid Y coord. The purpose of this variable triplet is to de-duplicate and match dig requests, so there's not even an explicit check for validity, only for "equals the last set sent by the client". I support this change, as it makes the value obviously invalid.